agent/grpc-external/services/resource/write_status_test.go

```diff
@@ -49,7 +49,7 @@ func TestWriteStatus_ACL(t *testing.T) {
 			mockACLResolver.On("ResolveTokenAndDefaultMeta", mock.Anything, mock.Anything, mock.Anything).
 				Return(tc.authz, nil)
 			server.ACLResolver = mockACLResolver
-			demo.Register(server.Registry)
+			demo.RegisterTypes(server.Registry)
```

Comment:
This looks to have been missed by previous commits to change  demo.Register -> demo.RegisterTypes